packages/react-ui/src/app/connections/create-edit-connection-dialog.tsx

```diff
@@ -145,7 +145,6 @@ const CreateOrEditConnectionDialog = React.memo(
         return appConnectionsApi.upsert(formValues);
       },
       onSuccess: (connection) => {
-        setOpen(false);
```

Comment:
this is the cause for the `window.activepieces.connect` to resolve the promise prematurely